eng/empty.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <!-- This project exists purely to work around Arcade
+       behavior. Arcade knows how to restore tasks needed for
+       publishing to blob feeds, but it requires a project file that
+       imports nuget restore targets to be specified for this to
+       work. See details in
+       https://github.com/dotnet/arcade/commit/f657be5cb7cd4920334dd9162173b131211a1e17#r31728598.
+  -->
+
+  <PropertyGroup>
```

Comment:
Is this needed?

Reply:
This is here to satisfy the expectation of the eng/native/build-commons.sh script that the project exists. It can be excluded if everyone remembers to always pass `-skipconfigure`. Maybe I can change the higher level scripts to always pass this and then eliminate this project and the targets file. Alternatively, we keep it and try to incorporate the version information file it generates (I would do that post merge).